### ISSUE TITLE
Counting roots. 

### ISSUE THREAD

**Issue by vutruong99** [SmartRoot/SmartRoot#18]
Hi guys I'm new to SmartRoot and I am helping a team analyzing roots. I can count roots manually by tracing them, but I was wondering if there is a feature to automatically count the number of roots like how ImageJ can count the number of cells. Sorry if this is a bad question :(

**Comment by guillaumelobet** [comment]
no, you have the trace the roots to have a count. If you want to have a more automated way of doing so, check out the www.plant-image-analysis.org website, which references many root image analysis tools.

**Comment by vutruong99** [comment]
Thank you very much for your answer! :)
